Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "v8_rs"
 version = "0.3.0"
 edition = "2021"
-license = "Redis Source Available License 2.0 (RSALv2) or the Server Side Public License v1 (SSPLv1)"
+license = "LicenseRef-RSALv2 OR SSPL-1.0"
```

Comment:
Can we avoid changing this? I believe that if not we probably need approval from the legal team.